src/video.ts

```diff
@@ -34,6 +37,8 @@ const parseVideo = (element: BlockElement, atoms?: Atoms): Option<Video> => {
 		posterUrl,
 		videoId,
 		duration: duration?.toNumber(),
+		atomId: id,
+        title
```

Comment:
Indent here should be inline with the above

Reply:
Strange that the linter didn't pick this up 🤔